package.json

```diff
@@ -28,21 +28,25 @@
     "rimraf": "^2.6.1",
     "ts-loader": "^2.0.1",
     "typescript": "^2.1.6",
-    "zone.js": ">=0.7.2"
+    "@angular/common": "4.0.2",
+    "@angular/compiler": "4.0.2",
+    "@angular/core": "4.0.2",
+    "@angular/platform-browser": "4.0.2",
+    "@angular/platform-browser-dynamic": "4.0.2",
+    "rxjs": "^5.0.3",
+    "zone.js": ">=0.8.5"
   },
   "dependencies": {
-    "@angular/common": ">=2.0.0",
-    "@angular/compiler": ">=2.0.0",
-    "@angular/core": ">=2.0.0",
-    "@angular/platform-browser": ">=2.0.0",
-    "@angular/platform-browser-dynamic": ">=2.0.0",
     "bootstrap": "^3.3",
     "eonasdan-bootstrap-datetimepicker": ">=4.17.22 <5.0.0",
     "@types/eonasdan-bootstrap-datetimepicker": "^4.17.22",
     "@types/jquery": "^2.0.33",
     "moment": "^2.10",
     "moment-timezone": "^0.4.0",
-    "jquery": "^1.8.3 || ^2.0",
-    "rxjs": "^5.0.1"
+    "jquery": "^1.8.3 || ^2.0"
+  },
+  "peerDependencies": {
+    "@angular/common": ">=2.3.0",
```

Comment:
Ok so finally.

I think these should be
```
"peerDependencies": {
    "@angular/common": ">=2.0.0",
    "@angular/core": ">=2.0.0"
  }
```
since it might be a blocker if someone uses older version.

And then I can accept your PR and deploy.

Reply:
done